pulsar-broker/pom.xml

```diff
@@ -119,6 +119,34 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
I think the issue is that we're not specifying ZK in the `<dependencyManagement>` section like most other dependencies. Right now we only have the `tests` ZK artifact defined there: https://github.com/apache/incubator-pulsar/blob/master/pom.xml#L168 

Once we set it in the `dependencyManagement` it will override the ZK version for all the modules in the project (and transitive dependencies)

Reply:
Ah, will move it there.